backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -43,7 +43,15 @@ public static IQueryable<Entity.Views.Property> GenerateQuery(this PimsContext c
                 query = query.Where(p => !p.IsSensitive);
 
             if (filter.PropertyType.HasValue)
-                query = query.Where(p => p.PropertyTypeId == filter.PropertyType);
+                query = query.Where(p => p.PropertyTypeId == filter.PropertyType.Value);
+
+            if (filter.BareLandOnly == true)
```

Comment:
Seems to work, but want confirmation that this is ok!

Reply:
Looks correct, the real test is the result :P